.ci/windows-steps.yaml

```diff
@@ -138,7 +136,7 @@ steps:
 # Run tests via ctest.
 - bash: |
     cd build/tests
-    CTEST_OUTPUT_ON_FAILURE=1 ctest -T Test -C Release .
+    CTEST_OUTPUT_ON_FAILURE=1 ctest -T Test .
```

Comment:
Looks like you removed the serialization package, in which case you also have to remove:

https://github.com/mlpack/models/blob/47ba8165a535e0f3f9d50481965f4e16653da19e/CMakeLists.txt#L172

Reply:
Hey, I did not remove serialization i only removed boost unit testing.